.travis.yml

```diff
@@ -148,7 +125,7 @@ jobs:
   #
   ################################################################################
   - stage: build dependencies 🔩
-    if: type != pull_request AND branch = master
+    if: type != pull_request
```

Comment:
Now these will trigger on feature branches too?

Reply:
No, because of -> https://github.com/input-output-hk/cardano-wallet/pull/290/files#diff-354f30a63fb0907d4ad57269548329e3L27

We already have a condition that applies to ALL jobs. So it was slightly redundant to have these condition, except maybe that... we might not want to do any deploy stage with bors!

Reply:
@Anviking Okay, I went for a slightly different approach and instead, allowed bors to run on PR builds (which are much faster in the end so better fits the problem here). This makes bors do only what's necessary to ensure PRs build correctly, and then, still leave to travis to perform deployment and other heavier tasks when stuff reaches `master`.